dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.41/V2_41_1000__RenameTrackedEntiyCommentToNote.sql

```diff
@@ -0,0 +1,50 @@
+-- rename trackedentitycomment to note
```

Comment:
the comments are not needed IMHO, as they are repeating the sentence from the `alter table` which is as readable 😄